app/controllers/admin/stats_controller.rb

```diff
@@ -0,0 +1,23 @@
+class Admin::StatsController < ApplicationController
+
+  before_filter :admin_only
+
+  def index
+    @graph_dates = []
+    d = 3.months.ago.beginning_of_week
+    while d < Time.now
+      @graph_dates << d
+      d = d + 7.days
+    end
+    if params[:model] == "invitations"
+      @invitations_by_status = {:unsent => Invitation.unsent.count, :unredeemed => Invitation.unredeemed.count, :redeemed => Invitation.redeemed.count}
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Space inside { missing.<br>Space inside } missing.